app/controllers/api/migration_analytics_manifest_controller.rb

```diff
@@ -0,0 +1,34 @@
+def parse_json_if_valid(json)
+    return JSON.parse(json)
+  rescue JSON::ParserError => e
```

Comment:
If you're not doing anything with the exception object then you don't need to assign to `=> e`

Reply:
Ah ok. Wasn't sure if that was just a required part of the syntax for `rescue`